semantic/tasks/rtl/watch.js

```diff
@@ -0,0 +1,221 @@
+/*******************************
+           Watch Task
+*******************************/
+
+var
+  gulp         = require('gulp'),
+
+  // node deps
+  console      = require('better-console'),
+  fs           = require('fs'),
+
+  // gulp deps
+  autoprefixer = require('gulp-autoprefixer'),
+  chmod        = require('gulp-chmod'),
+  clone        = require('gulp-clone'),
+  gulpif       = require('gulp-if'),
+  less         = require('gulp-less'),
+  minifyCSS    = require('gulp-clean-css'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)